src/functions.cpp

```diff
@@ -14,13 +13,7 @@ void sort(std::vector<int> & unsorted) {
     std::sort(begin(unsorted), end(unsorted));
 }
 
-int min(std::vector<int> const & values) {
-    int minimum = std::numeric_limits<int>::max();
-    for (const auto & value : values) {
-        if (value < minimum) {
-            minimum = value;
-        }
-    }
-    return minimum;
```

Comment:
Why did you deleted min function? It should remain here (as a part of master branch)

Reply:
My mistake!